bayesian/api_v1.py

```diff
@@ -737,9 +738,23 @@ def post():
         github_url = request.form.get("github_url")
         ref = request.form.get('github_ref')
         user_email = request.headers.get('UserEmail')
+        scan_repo_url = request.headers.get('scanRepoUrl')
+        print('Email ' + user_email)
```

Comment:
Use logger statements as in - https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/bayesian/api_v1.py#L761

Reply:
I would not even want to log this information in our logs. especially user_email